src/main/java/org/apache/gossip/manager/random/RandomGossipManager.java

```diff
@@ -42,7 +43,14 @@ public static ManagerBuilder newBuilder() {
 
     private ManagerBuilder() {}
 
+    private void checkNotNull(Object arg, String msg) {
```

Comment:
@doellerbe The apache-way is to comment in apache systems, jira, ML. I realize it is a pain when using github. I suggested changing to IllegalArgumentException in the ticket. Commenting here spams the ML so we probably do not want that.